classes/module/Module.php

```diff
@@ -3224,6 +3231,22 @@ public function addOverride($classname)
         return true;
     }
 
+    private function createOverrideDirectory(string $directoryPath): void
+    {
+        // Create directory (with recurrence)
+        @mkdir($directoryPath, FileSystem::DEFAULT_MODE_FOLDER, true);
+
+        // Copy index.php to each directory
+        $directoryOverride = _PS_ROOT_DIR_ . DIRECTORY_SEPARATOR . 'override' . DIRECTORY_SEPARATOR;
```

Comment:
duplicate from line 3167, can be something you refactor .

You're also able to use SplFileInfo object through Symfony FS, in order to be able to properly handle parent directories, it also has a better way to copy: https://github.com/symfony/filesystem/blob/4.4/Filesystem.php#L41

Reply:
Fixed